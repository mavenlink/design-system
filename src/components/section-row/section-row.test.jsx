import React from 'react';
import { render, screen } from '@testing-library/react';
import SectionRow from './section-row.jsx';

describe('Section Row', () => {
  it('renders a section-row component', () => {
    render(<SectionRow />);
    expect(document.body).toMatchSnapshot();
  });

  it('can have child elements', () => {
    render(<SectionRow><span>Test Label</span></SectionRow>);
    expect(screen.getByText('Test Label').tagName).toEqual('SPAN');
  });
});
