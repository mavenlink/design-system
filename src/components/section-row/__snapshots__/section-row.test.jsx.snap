// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Row renders a section-row component 1`] = `
<body>
  <div>
    <div
      class="section-row"
    />
  </div>
</body>
`;
